PWGCF/MultiparticleCorrelations/Tasks/multiparticle-correlations-ab.cxx

```diff
@@ -25,28 +25,29 @@ using namespace o2::framework;
 
 // *) Run 3:
 using EventSelection = soa::Join<aod::EvSels, aod::Mults, aod::CentFT0Ms, aod::CentFV0As, aod::CentNTPVs>;
-using CollisionRec = soa::Join<aod::Collisions, EventSelection>::iterator;
+using CollisionRec = soa::Join<aod::Collisions, EventSelection>::iterator; // use in json "isMC": "true" for "event-selection-task"
```

Comment:
Shouldn't the comment go to the next line?

Reply:
I still do not have uniform style when it comes to commenting, but I will get to it eventually.